.github/workflows/build.yml

```diff
@@ -50,13 +50,13 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --workspace --all-targets --all-features
+          args: --workspace --lib --bins --tests --examples --all-features
```

Comment:
avoid building `benches` directory

Reply:
That will result in a fact that future PRs can break benches and we will not notice. Maybe add a separate CI to not run benches on 1.41.1 but still run on other versions?

Reply:
that's true. fixed